src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -27,6 +28,8 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         requireNonNull(word);
 
         String preppedWord = word.trim();
+        checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
+        checkArgument(preppedWord.split("\\s+").length == 1, "Word parameter should be a single word");
```

Comment:
Do you think it would be more readable if these two strings are abstracted to `static final` variables?